nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs

```diff
@@ -245,25 +262,30 @@ public static IEnumerable<Dependency> GetTopLevelPackageDependencyInfos(Immutabl
     /// <summary>
     /// Given an MSBuild string and a set of properties, returns our best guess at the final value MSBuild will evaluate to.
     /// </summary>
-    /// <param name="msbuildString"></param>
-    /// <param name="propertyInfo"></param>
-    /// <returns></returns>
-    public static string GetRootedValue(string msbuildString, Dictionary<string, string> propertyInfo)
+    public static EvaluationResult GetEvaluatedValue(string msbuildString, Dictionary<string, string> propertyInfo, params string[] propertiesToIgnore)
```

Comment:
MSBuild properties are case insensitive.  `propertiesToIgnore` should probably be internally converted to a `HashSet<string>` with a case insensitive comparer.

Reply:
@brettfo Good catch. Updated!